.circleci/create_circleci_config.py

```diff
@@ -154,6 +154,7 @@ def job_name(self):
         "sudo apt-get -y update && sudo apt-get install -y libsndfile1-dev espeak-ng time",
         "pip install --upgrade pip",
         "pip install .[sklearn,torch,testing,sentencepiece,torch-speech,vision,timm]",
+        "pip install natten",
```

Comment:
I'll ping @LysandreJik and @sgugger here regarding this extra dependency. 

Ideally we don't add additional dependencies for models, for instance Deformable DETR also relies on a custom kernel and it's just a folder in the [model folder](https://github.com/huggingface/transformers/tree/main/src/transformers/models/deformable_detr).